src/components/GFWFishingEffort.tsx

```diff
@@ -4,38 +4,36 @@ import {
   ResultsCard,
   ClassTableColumnConfig,
   useSketchProperties,
+  ClassTable,
+  SketchClassTable,
 } from "@seasketch/geoprocessing/client-ui";
 import {
   ReportResult,
-  ReportResultBase,
   toNullSketchArray,
   flattenBySketchAllClass,
   metricsWithSketchId,
   NullSketch,
   NullSketchCollection,
   Metric,
   MetricGroup,
+  toPercentMetric,
 } from "@seasketch/geoprocessing/client-core";
 import cloneDeep from "lodash/cloneDeep";
 import { Trans, useTranslation } from "react-i18next";
 import project from "../../project";
-import {
-  getPrecalcMetrics,
-  toPercentMetric,
-} from "../../data/bin/getPrecalcMetrics";
 import { GeoProp } from "../types";
-import { getGeographyById } from "../util/getGeographyById";
-import { ClassTable } from "../util/ClassTable";
-import { SketchClassTable } from "../util/SketchClassTable";
 
 export const GFWFishingEffort: React.FunctionComponent<GeoProp> = (props) => {
   const [{ isCollection }] = useSketchProperties();
   const { t } = useTranslation();
   const metricGroup = project.getMetricGroup("gfwValueOverlap", t);
-  const precalcTotals: Metric[] = getPrecalcMetrics(
+  const curGeography = project.getGeographyById(props.geographyId, {
+    fallbackGroup: "default-boundary",
+  });
+  const precalcTotals: Metric[] = project.getPrecalcMetrics(
     metricGroup,
     "sum",
-    props.geographyId
+    curGeography.geographyId
   );
```

Comment:
This is now the standard incantation for every geoprocessing function.  It extracts one geographyId from geographyIds param if it exists, and falls back to the default geography otherwise.  There should be one geography assigned as the `default-geography` for every project.  By default this is the `world` for a basic project and `eez` geography for an eez project.